Traceback (most recent call last):
  File "simclr_transfer.py", line 441, in <module>
    flops, params = profile(model, inputs=(torch.randn(1, 3, 32, 32).cuda(),))
  File "/mnt/home/renjie3/miniconda3/envs/simclr/lib/python3.7/site-packages/thop/profile.py", line 188, in profile
    model(*inputs)
  File "/mnt/home/renjie3/miniconda3/envs/simclr/lib/python3.7/site-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/mnt/ufs18/home-145/renjie3/Documents/unlearnable/Unlearnable-Examples/model.py", line 33, in forward
    x = self.f(x)
  File "/mnt/home/renjie3/miniconda3/envs/simclr/lib/python3.7/site-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/mnt/home/renjie3/miniconda3/envs/simclr/lib/python3.7/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/mnt/home/renjie3/miniconda3/envs/simclr/lib/python3.7/site-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/mnt/home/renjie3/miniconda3/envs/simclr/lib/python3.7/site-packages/torch/nn/modules/activation.py", line 102, in forward
    return F.relu(input, inplace=self.inplace)
  File "/mnt/home/renjie3/miniconda3/envs/simclr/lib/python3.7/site-packages/torch/nn/functional.py", line 1134, in relu
    result = torch.relu_(input)
RuntimeError: CUDA error: no kernel image is available for execution on the device
/var/lib/slurmd/job103499541/slurm_script: line 44: [1: command not found
